geo-data-server/src/test/java/com/powsybl/geodata/server/GeoDataControllerTest.java

```diff
@@ -77,6 +70,14 @@ public void setUp() {
         MockitoAnnotations.initMocks(this);
     }
 
+    @Autowired
+    private Cassandra cassandra;
+
+    @After
```

Comment:
Why no @AfterClass ?

Reply:
Actually, you should not do this by yourself. `@EmbeddedCassandra` stops Cassandra by itself.

Reply:
If you have any issues when Cassandra still running between the tests, you have to add `@DirtiesContext` annotation. 
This happens because `Spring Test` framework caches the application context and one of the beans is `Cassandra`. 

You have actually two options:
 -  Use random ports  (default behaviour) 
-  Add @DirtiesContext

Reply:
Indeed it now works without having to stop it. Thanks a lot.